packages/core/src/compile.ts

```diff
@@ -171,34 +161,37 @@ function serialiseTranslationRuntime(
       )}) => ${formatReturn}`;
     }
 
-    translationsType[encodeBackslash(key)] = translationFunctionString;
+    translationsType[key] = translationFunctionString;
```

Comment:
Was this previously not needed or what has changed to not need it now?

Reply:
It's now handled here: https://github.com/seek-oss/vocab/blob/d4fa3b0a1829862168e6f3099ce749b9bc2388e0/packages/core/src/compile.ts#L122-L126